Automattic/wp-calypso
# Sites page: Add new empty state CTAs

Related to https://github.com/Automattic/dotcom-forge/issues/2365

## Proposed Changes

This adds a new empty state component to the sites page, replacing the old one (after fetching sites). Screenshot:

![page](https://user-images.githubusercontent.com/26530524/241073224-13bc4624-928c-4f1d-af1a-097d847c4b7b.png)

## Testing Instructions

Make sure that `hosting-onboarding-i2` flag is activated and run this branch on an account with zero sites

1. Check that both CTAs appear and they link to `/setup/new-hosted-site` and `/setup/import-focused`, respectively;
2. Check that hovering/focusing the CTAs activate the sparkling animation, and moving the mouse out of the link removes the sparkles after a while;
3. Check that turning off the flag changes the first CTA to `/start`.
